core/templates/dev/head/css/oppia.css

```diff
@@ -31,6 +31,35 @@
   100% { opacity: 1; }
 }
 
+@keyframes fadein {
+  0% { opacity: 0; }
+  100% { opacity: 1; }
+}
+
+/* Firefox < 16 */
+@-moz-keyframes fadein {
+  0% { opacity: 0; }
+  100% { opacity: 1; }
+}
+
+/* Safari, Chrome and Opera > 12.1 */
```

Comment:
You're right, I don't think we need to support very old versions because it looks like not many people are using it. Latest and relatively older ones should be okay I think.